stubs/pyxdg/xdg/IconTheme.pyi

```diff
@@ -0,0 +1,59 @@
+from _typeshed import Unused
+from collections.abc import Collection
+from os import PathLike
+from typing import AnyStr
+
+from xdg.IniFile import IniFile
+
+basedir: Unused  # This is something that shouldn't be included since it's
+
+class IconTheme(IniFile):
+    def __init__(self) -> None: ...
+    dir: str
+    name: str
+    comment: str
+    directories: list[str]
+    type: str
+    def parse(self, file: PathLike[AnyStr]) -> None: ...  # type: ignore[override]
+    def getDir(self) -> str: ...
+    def getName(self) -> str: ...
+    def getComment(self) -> str: ...
+    def getInherits(self) -> list[str]: ...
+    def getDirectories(self) -> list[str]: ...
+    def getScaledDirectories(self) -> list[str]: ...
+    def getHidden(self) -> bool: ...
+    def getExample(self) -> str: ...
+    def getSize(self, directory: PathLike[AnyStr]) -> int: ...
```

Comment:
It doesn't look like these would work well with bytes. Let's use str instead of AnyStr.

Reply:
Bytes paths are often a bit of an afterthought and APIs only really work thoroughly for them if the API author has been careful to preserve support. It sounds like pyxdg didn't really do that. Few real-world users are likely to pass bytes paths, so it's probably not a problem for them.